Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/lucene/impl/LuceneUtility.java

```diff
@@ -648,6 +707,14 @@ private void buildTokenizedOrWildCardQuery(LuceneField luceneFieldAnnotation, St
     private void createWildCardSearchQuery(LuceneField luceneFieldAnnotation, String searchString,
             BooleanQuery.Builder queryBuilder, Occur occur) {
         if (luceneFieldAnnotation.isWildCardSearchEnabled()) {
+            /*StringBuilder sb = new StringBuilder(searchString.length());
```

Comment:
Please remove this comments too

Reply:
Ya, have done that and committed already.